source/channels.yaml

```diff
@@ -73,6 +73,43 @@
         '403':
           description: User does not belong to the team
 
+  '/teams/{team_id}/channels/view':
+    post:
+      tags:
+        - channels
+      summary: View a channel
+      description: Perform all the actions involved in viewing a channel. This includes marking channels as read, clearing push notifications, and updating the active channel.
+      parameters:
+        - in: path
+          name: team_id
+          description: Team ID of the channel to update
+          required: true
+          type: string
+        - in: body
+          name: body
+          description: Paremeters affecting how and which channels to view.
+          required: true
+          schema:
+            type: object
+            required:
+              - channel_id
+            properties:
+              channel_id:
+                type: string
+                description: The channel ID that is being viewed. Use a blank string to indicate that all channels have lost focus.
+              prev_channel_id:
+                type: string
+                description: The channel ID of the previous channel, used when switching channels. Providing this ID will cause push notifications to clear on the channel being switched to.
+              time:
+                type: string
+                description: The Unix time, in milliseconds, that the channel was viewed at. Provide a non-zero value for this parameter to use a manual time. If this parameter is zero or not included, the server will use the current time.
+
+      responses:
+        '200':
+          description: Channel viewed successfully.
+          schema:
+            $ref: '#/definitions/StatusOK'
```

Comment:
What do you use the time for on the RN app? I see it getting stored, but where is it used?